### PR TITLE
PICARD-1226: Fix font size issues occuring while resizing plugin option page. 

### PR DESCRIPTION
**Pull request by vishichoudhary**
# Summary

* This is a…
    * [x] Bug fix
    * [ ] Feature addition
    * [ ] Refactoring
    * [ ] Minor / simple change (like a typo)
    * [ ] Other
* **Describe this change in 1-2 sentences**:

# Problem
While resizing the plugin option page, font size in plugin names changes.

* JIRA ticket (_optional_): [PICARD-1226](https://tickets.metabrainz.org/browse/PICARD-1226)

# Solution
Just force set the style sheet for qtreewidget.

# Action

before
bug1

after 
solve

### CODE REVIEW

**Comment by vishichoudhary** [comment]
solution verified on windows 10

File: picard/ui/options/plugins.py [hunk 1]
```diff
@@ -86,6 +86,8 @@ def __init__(self, parent=None):
         super().__init__(parent)
         self.ui = Ui_PluginsOptionsPage()
         self.ui.setupUi(self)
+        #fix for PICARD-1226, QT bug around
```
Comment (zas) [inline picard/ui/options/plugins.py]: Can you just add links to related QT bugs for future reference ?

Comment (vishichoudhary) [reply picard/ui/options/plugins.py]: Ok i will add one.. :-)

**Comment by vishichoudhary** [comment]
@zas added link to qt bug

**Comment by zas** [comment]
@vishichoudhary : can you try something ? without this patch, comment out https://github.com/metabrainz/picard/blob/master/picard/tagger.py#L121-L123 and tell me if it changes anything regarding this issue on win.

**Comment by vishichoudhary** [comment]
@zas https://github.com/metabrainz/picard/blob/master/picard/tagger.py#L121-L123 this line has no connection with this issue. I commented that line and without this path, i have still PICARD-1226 issue.

**Comment by zas** [comment]
> I commented that line and without this path, i have still PICARD-1226 issue.

Ok thanks, i wanted to be sure it wasn't caused by the other workaround...

**Pull request merged**